edx_django_utils/monitoring/internal/middleware.py

```diff
@@ -285,6 +286,19 @@ def __call__(self, request):
 
         response = self.get_response(request)
 
+        # .. setting_name: COOKIE_PREFIXES_TO_REMOVE
+        # .. setting_default: []
+        # .. setting_description: A list of cookie prefixes. Any cookie starting with a prefix in this list
+        # will be manually removed from responses (i.e. set to expire in the past)
+        deprecated_cookie_prefixes = getattr(settings, "COOKIE_PREFIXES_TO_REMOVE", [])
+        yesterday = datetime.utcnow() - timedelta(days=1)
+        for cookie_name in request.COOKIES.keys():
```

Comment:
What if we switched the two for-loops? Then if there are no prefixes to iterate over, we don't have to loop over the cookies either.